README.md

```diff
@@ -159,11 +159,23 @@ $ curl -F "image=@assets/dog-human.jpg" -XPOST http://127.0.0.1:5000/model/predi
 The optional `threshold` parameter is the minimum `probability` value for predicted labels returned by the model.
 The default value for `threshold` is `0.7`.
 
-### 4. Development
+### 4. Run the Notebook
```

Comment:
You will need to update the contents list section at https://github.com/IBM/MAX-Object-Detector/pull/44/files#diff-04c6e90faac2675aa89e2176d2eec7d8R72 accordingly

Reply:
Fixed in 97937c7.